tools/metaquantome/macros.xml

```diff
@@ -1,10 +1,9 @@
 <macros>
-    <token name="@VERSION@">0.99.3</token>
-    <token name="@GVERSION@">0.99.3.0</token>
+    <token name="@VERSION@">0.99.4a0</token>
+    <token name="@GVERSION@">0.99.4a00</token>
```

Comment:
4a0.0?

Reply:
that's not allowed by `planemo lint`, so i changed it to `0.99.4a0-0`. It'll get simpler once we test - and we'll hopefully release a v 1.0!